query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "      <th></th>\n",
+       "      <th>loc_id</th>\n",
+       "      <th>status</th>\n",
+       "      <th>name</th>\n",
+       "      <th>city</th>\n",
+       "      <th>country</th>\n",
+       "      <th>org_id</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1048</td>\n",
+       "      <td>active</td>\n",
+       "      <td>PCP83180</td>\n",
+       "      <td>EVERETT</td>\n",
+       "      <td>US</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1064</td>\n",
+       "      <td>active</td>\n",
+       "      <td>PHYSICAL THERAPY AND FITNESS CENTER OF RAYNHAM...</td>\n",
+       "      <td>RAYNHAM</td>\n",
+       "      <td>US</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>112</td>\n",
+       "      <td>active</td>\n",
+       "      <td>PCP77</td>\n",
+       "      <td>PITTSFIELD</td>\n",
+       "      <td>US</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>1236</td>\n",
+       "      <td>active</td>\n",
+       "      <td>PCP129774</td>\n",
+       "      <td>REVERE</td>\n",
+       "      <td>US</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1292</td>\n",
+       "      <td>active</td>\n",
+       "      <td>PCP144578</td>\n",
+       "      <td>BILLERICA</td>\n",
+       "      <td>US</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  loc_id  status                                               name  \\\n",
+       "0   1048  active                                           PCP83180   \n",
+       "1   1064  active  PHYSICAL THERAPY AND FITNESS CENTER OF RAYNHAM...   \n",
+       "2    112  active                                              PCP77   \n",
+       "3   1236  active                                          PCP129774   \n",
+       "4   1292  active                                          PCP144578   \n",
+       "\n",
+       "         city country org_id  \n",
+       "0     EVERETT      US   None  \n",
+       "1     RAYNHAM      US   None  \n",
+       "2  PITTSFIELD      US   None  \n",
+       "3      REVERE      US   None  \n",
+       "4   BILLERICA      US   None  "
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_location AS\n",
+    "      SELECT L.id AS loc_id, L.status, L.name, L.address.city,\n",
```

Comment:
For Location, would it be feasible to add long/lat coordinates. Assumption is that if they were here it would potentially facilitate doing mapping based visualizations

* position
*   longitude
*   latitude
*   altitude